core/block.go

```diff
@@ -124,38 +72,40 @@ func VerifyBlockHash(b *Block, network utils.Network) (*BlockCommitments, error)
 		return nil, err
 	}
 
-	metaInfo := NetworkBlockHashMetaInfo(network)
-	unverifiableRange := metaInfo.UnverifiableRange
-	for _, fallbackSeq := range []*felt.Felt{&felt.Zero, metaInfo.FallBackSequencerAddress} {
-		var overrideSeq *felt.Felt
-		if b.SequencerAddress == nil {
-			overrideSeq = fallbackSeq
-		}
+	metaInfo := network.BlockHashMetaInfo
+	if metaInfo != nil {
```

Comment:
We should still calculate the block hash if meta info isn't set. We can have a default meta info.

Reply:
Reverted this change.